crates/sncast/tests/e2e/account/helpers.rs

```diff
@@ -0,0 +1,67 @@
+use indoc::indoc;
+use tempfile::{tempdir, TempDir};
+use tokio::fs::File;
+use tokio::io::AsyncWriteExt;
+
+#[must_use]
+pub async fn create_tempdir_with_empty_json(file_name: &str) -> TempDir {
```

Comment:
What about making this function sync?

Reply:
It used to be async before I moved it into the common module, however I don't insist on it

Reply:
Please make it sync then

Reply:
I mean - it's not my function. I just moved an already existing one into a common module to use it widely